docs/SetupDatabaseForAKS.md

```diff
@@ -0,0 +1,105 @@
+Scalar schema tool abstracts Request Unit of Cosmos DB with RU. So please set the appropriate value.
+
+#### Autoscale
+
+Autoscale provisioned throughput in Azure Cosmos DB allows you to scale the throughput (RU/s) of your database or container automatically and instantly.
+The throughput is scaled based on the usage, without impacting the availability, latency, throughput, or performance of the workload.
+By default, the scalardl schema tool enables autoscale of RU for all tables: RU is scaled in or out between 10% and 100% of a specified RU depending on a workload.
+
+### Monitor Cosmos DB
+
+By default, monitoring is enabled on Cosmos DB.
+
+More information can be found in [Monitor Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/monitor-cosmos-db) documentation.
+
+## Azure Database for MySQL
+
+Azure Database for MySQL is a fully managed MySQL database provided by Azure. You can create a MySQL database by selecting a database instance and storage according to your configurations. The IOPS is calculated based on the compute instance and storage you choose.
+In this section, you will create an Azure Database for MySQL.
+
+### Recommendations
+
+* You should choose Flexible or Single server deployment for Azure Database for MySQL. You can check the [official documentation](https://docs.microsoft.com/en-us/azure/mysql/select-right-deployment-type) to learn more about the differences between Flexible and Single server deployments.
+* You should select a deployment that suits your IOPS requirements.
+    * For Single server deployments, IOPS is defined on the amount of storage provisioned(3 IOPS/GB).
+    * For Flexible server deployments, additional IOPS can be provisioned and the maximum threshold will be defined by the storage and the database instance chosen.
+
+### Steps
+
+You can follow [official documentation](https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal) to create an Azure Database for MySQL server for Single server deployment.
+For creating a flexible server deployment for Azure Database for MySQL, you can follow this [official guide](https://docs.microsoft.com/en-us/azure/mysql/flexible-server/quickstart-create-server-portal).
+
+### Scale Performance
+
+**Scaling storage**:- You can enable the `storage auto grow` feature to automatically increase storage when it reaches a threshold without impacting the workload.
+
+You can scale resources based on the deployment you choose. For Single server deployment, you can refer to the [official documentation](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#scale-resources) for scaling resources.
+For Flexible server deployments, additional IOPS can be provisioned. You can read more about it [here](https://docs.microsoft.com/en-gb/azure/mysql/flexible-server/concepts-compute-storage#iops).
```

Comment:
Sentence and reference are inconsistent.

Reply:
Updated section and links.